doc/source/create-k8s-cluster.rst

```diff
@@ -163,6 +169,121 @@ Setting up Kubernetes on Microsoft Azure Container Service (ACS)
+   on page 13. Clicking on the 'Launch' button of each figure will direct your browser
+   to the cloud formation console, with the default values pre-filled by Heptio.
+   We recommend ``Option 1``, which will create a new set of resources
+   on AWS to run Kubernetes.
+
+   After clicking the ``Option 1`` button, you'll be taken to an AWS page with a field already
+   chosen under "Choose a template". Simply hit "Next".
+
+   **Enter AWS instance information (page 1)**: On this page you'll need to
+   fill in the following required fields:
+
+   * ``Stack Name`` can be anything you like.
+   * ``Availability Zone`` is related to the location of the AWS
+     resources. Choose an AWS location close to your physical location or
+     any other desired AWS location.
+   * ``Admin Ingress Location`` defines the locations from which you
+     can access this cluster as an administrator. Enter ``0.0.0.0/0``
+     for the most permissive approach.
+   * ``SSH Key`` is a dropdown list of keys attached to your account.
+     The one you created in Step 1 should be listed here. This will allow
+     you to SSH into the machines if you desire.
+   * ``Node Capacity`` defines the number of machines you've got available.
+     This will depend on the ``Instance Type`` that you choose. E.g., if you
+     want each user to have 2GB and you expect 10 users, choose a combination
+     of ``Instance Type`` and ``Node Capacity`` that meets this requirement.
+   * ``Instance Type`` defines what kind of machine you're requesting. See
+     this `list of instance types with Amazon <https://aws.amazon.com/ec2/instance-types/>`_
+     as well as this list of `pricing for each instance type <https://aws.amazon.com/ec2/pricing/on-demand/>`_.
+   * ``Disk Size`` corresponds to the hard disk for each node. Note that this is
+     different from the disks that users will use for their own notebooks/data.
+     This disk should be large enough to contain the size of any Docker
+     images you're serving with the JupyterHub.
+   * ``Instance Type (Bastion Host)`` corresponds to a computer that allows
+     for easy SSH access to your Kubernetes cluster. This does not need to
+     be a fancy computer. You may leave these as defaults. For more information
+     on the Bastion Host, `see here <http://docs.aws.amazon.com/quickstart/latest/linux-bastion/architecture.html>`_.
+
+   **Enter AWS instance information (page 2)**: On the second page you may leave
+   all of these fields as is or customize as you wish. When done, hit ``Next``. Then
+   confirm and hit ``Next`` once more.
+
+   AWS will now create the computational resources defined in the Heptio
+   template (and according to the options that you chose). This will
+   take a few minutes.
+
+   To see the status of the resources you've requested,
+   see the ``CloudFormation`` page. You should see two stacks being created,
+   each will have the name you've requested. When they're done creating,
```

Comment:
I would mention that this can take approximately 10-20 minutes. And that you cannot continue with the next steps until this elapses (in other words, "go refill your coffee").